invenio_users_resources/services/results.py

```diff
@@ -58,5 +58,5 @@ def last_modified(self):
 
     @property
     def max_age(self):
-        """Get last modified date for the response."""
```

Comment:
minor: would be nice to have a reference in the doc about this value's unit (seconds, ms, etc)